remoteworkitem/trackerquery_repository.go

```diff
@@ -61,7 +61,7 @@ func (r *GormTrackerQueryRepository) Load(ctx context.Context, ID string) (*app.
 		ID:        string(res.ID),
 		Query:     res.Query,
 		Schedule:  res.Schedule,
-		TrackerID: int(res.TrackerID)}
+		TrackerID: string(res.TrackerID)}
```

Comment:
If the MediaType is used as the model for the Repository, and the ID is string in the MediaType, why does the Repository take an uint?